### PR TITLE
Disable search in multiple select field for Other Options field when not selected

### LINKED ISSUE #3457: cloud build feature selection component is buggy
### Describe the bug

I have consistently reproduced buggy behavior from the "tag list" component that handles feature selection for cloud builds. 

A video of the problem is availabile here : https://youtu.be/Zmzu7pTbqzc

### To Reproduce

As shown in the video, remove some features and add one (in my case , servo). 
the feature does not appear but it is selected, some text remains and there's void space in the input. 

### Expected behavior

An example of expected behavior can be found here :

https://vuetifyjs.com/en/components/autocompletes/#usage

Select : 
- [x] clearable
- [x] chips
- [x] multiple

### Configurator version

10.9.0

### Flight controller configuration

```txt
N/A
```

### Add any other context about the problem that you think might be relevant here

_No response_

### PR DESCRIPTION
**Pull request by haslinghuis**
Fixes #3457

We should not be able to type here - this part is still there - but disabling also disable the search. Disabling on field exit seems to fix the issue jumping back from typing in Custom Defines to Other Options field.

image

### CODE REVIEW

**Comment by blckmn** [comment]
AUTOMERGE: (FAIL)
 - github identifies PR as mergeable -> FAIL
 - assigned to a milestone -> PASS
 - cooling off period lapsed -> FAIL
 - commit count less or equal to three -> PASS
 - `Don't merge` label NOT found -> PASS
 - at least one `RN:` label found -> PASS
 - `Tested` label found -> FAIL
 - assigned to an approver -> PASS
 - approver count at least three -> FAIL

**Comment by McGiverGim** [comment]
This disables the search? I use it a lot, each time I need to add some define.
I need to test this, but I like to have a search.

**Comment by haslinghuis** [comment]
@McGiverGim yes - but I only want to stop enabling typing in the field. Maybe Vue has a better way (as I don't want to extend with adapters).

**Comment by nerdCopter** [comment]
i also use the search, then click the filtered result.  
maybe better to clear the text after click, but unsure how to clear without clearing selections. :frowning_face:

**Comment by McGiverGim** [comment]
I don't remove the text. When you search, yes, it remains in the field, but it seems is not "really" there. It seems not to affect the build, only the selected ones.

**Comment by nerdCopter** [comment]
> It would be better if we could have a search box separate from the selected items as backspace will remove last selected.

i'd be okay with backspace deletion if the cursor could be moved with keyboard. arrow keys do not move cursor :(

**Review by nerdCopter — APPROVED** [review]
`fa95e911` seems to clear the typing upon field exit; approving.

**Review by chmelevskij — APPROVED** [review]

**Pull request merged**
